src/style.css

```diff
@@ -0,0 +1,46 @@
+.page__body {
+  margin: 0;
+}
+
+.page__block {
+  width: 100px;
+}
+
+.block {
+  display: flex;
+  flex-direction: column;
+}
+
+.block__stars {
+  padding-right: 4px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+/* Selects all childs from +x to last position */
+.stars--0 :nth-child(n + 1),
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 3),
+.stars--3 :nth-child(n + 4),
+.stars--4 :nth-child(n + 5) {
+  background-image: url(images/star.svg);
+}
```

Comment:
u can apply this rule just for stars__star

Reply:
@SanyaBratashchuk, hi

Yes, but it will load star.svg for every star even behind star-active.svg

So, there are 2 options:
1. for sake of page load performance, or
2. to simplify code base

Every option has its own tradeoff, but with which one should I stay?

Reply:
Applied the rule for `.stars__star`